lib/configstore/config-types.js

```diff
@@ -0,0 +1,296 @@
+// TODO: move to / use mapeo-schema
+
```

Comment:
@gmaclennan this file is a rough start to the schemas (partially pulling from mapeo-schema) that has a few questions about the schemas and how certain types of config should be handled